Sources/SendGridKit/Models/Personalization.swift

```diff
@@ -21,8 +21,8 @@ public struct Personalization: Codable {
     public var substitutions: [String: String]?
     
     /// A collection of key/value pairs following the pattern "key":"value" to substitute handlebar template data
-    public var dynamicTemplateData: [String: String]?
-    
+    public var dynamicTemplateData: DynamicTemplateData?
```

Comment:
This technically breaks public API correct? @tonyarnold

Reply:
Yes, it does, sadly.

Reply:
Alright we'll just have new major versions of Kit and the helper library.

Reply:
Sorry! Without this change, though, it's not really possible to use SendGrid's dynamic templates properly. 

Seems worth it? 😄